### PR TITLE
PAT-2019 Tune kubernetes autoscaler to avoid alerts

### PR DESCRIPTION
**Pull request by steverendell**
Also correct alert target and minor tweaks

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -8,8 +8,12 @@ repositories {
   maven("https://dl.bintray.com/gov-uk-notify/maven")
 }
```
Comment (steverendell) [inline build.gradle.kts:10]: bring in line with other apis

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -13,3 +13,6 @@ generic-service:
 
 cron:
   namespace: check-my-diary-prod
+
```
Comment (steverendell) [inline helm_deploy/values-prod.yaml:16]: without this, alerts would go to dps_alerts_non_prod

File: src/main/java/uk/gov/justice/digital/hmpps/cmd/api/config/QuartzConfiguration.kt [hunk 3]
```diff
@@ -155,7 +155,11 @@ class QuartzConfiguration(val applicationContext: ApplicationContext) {
   }
 
   @Bean
```
Comment (steverendell) [inline src/main/java/uk/gov/justice/digital/hmpps/cmd/api/config/QuartzConfiguration.kt:157]: formatting only

File: src/test/resources/application-test.yml [hunk 4]
```diff
@@ -14,13 +14,15 @@ spring:
 
   datasource:
     url: 'jdbc:h2:mem:cmd-api-db-test;MODE=PostgreSQL'
```
Comment (steverendell) [inline src/test/resources/application-test.yml:16]: deprecated properties

**Review by monskapinga — APPROVED** [review]

**Pull request merged**